### PR TITLE
Fix middleware repr

### PR DESCRIPTION
**Pull request by ppd0705**

### CODE REVIEW

File: starlette/middleware/__init__.py [hunk 1]
```diff
@@ -12,6 +12,6 @@ def __iter__(self) -> typing.Iterator:
 
     def __repr__(self) -> str:
         class_name = self.__class__.__name__
-        option_strings = ["{key}={value!r}" for key, value in self.options.items()]
+        option_strings = [f"{key}={value!r}" for key, value in self.options.items()]
         options_repr = ", ".join(option_strings)
-        return f"{class_name}({self.cls.__name__}{options_repr})"
+        return f"{class_name}({self.cls.__name__}, {options_repr})"
```
Comment (lovelydinosaur) [inline starlette/middleware/__init__.py]: Worth having a look at the failing test case... https://travis-ci.org/encode/starlette/jobs/660170172?utm_medium=notification&utm_source=github_status

Comment (ppd0705) [reply starlette/middleware/__init__.py]: Already fixed , I don't known  if this way is ok?

**Comment by lovelydinosaur** [comment]
Nicely done, yup.

**Comment by ppd0705** [comment]
> Nicely done, yup.

thanks

**Pull request merged**